internal/test/framework/framework.go

```diff
@@ -369,6 +391,19 @@ func (f *TestFramework) RetrieveArtifacts() {
 	}
 }
 
+// ApplyHybridOverlayPatch will enable the hybrid overlay on the cluster
+func (f *TestFramework) ApplyHybridOverlayPatch() error {
+	jsonPatch := []byte(`{"spec":{"defaultNetwork":{"ovnKubernetesConfig":{"hybridOverlayConfig":` +
+		`{"hybridClusterNetwork":[{"cidr":"10.132.0.0/14","hostPrefix":23}]}}}}}`)
+
+	_, err := f.OSOperatorClient.Networks().Patch("cluster", k8stypes.MergePatchType, jsonPatch)
```

Comment:
Is this patching the network CR or the network.operator config CR? How are they being differentiated?

Reply:
This is patching the network.operator config CR. They are differentiated as the network CR is accessed through the OSConfigClient and the network.operator is accessed through the OSOperatorClient